test/unit/sanitizer/_text_pelias_parser.js

```diff
@@ -282,11 +305,11 @@ module.exports.tests.text_parser = function (test, common) {
   cases.push(['Kasch', { subject: 'Kasch' }, true]);
   cases.push(['Kaschk', { subject: 'Kaschk' }, true]);
   cases.push(['Kaschk ', { subject: 'Kaschk' }, true]);
-  // cases.push(['Kaschk B', { subject: 'Kaschk' }, true]); // jitter issue
-  cases.push(['Kaschk Be', { subject: 'Kaschk' }, true]);
-  // cases.push(['Kaschk Ber', { subject: 'Kaschk' }, true]); // jitter issue
-  // cases.push(['Kaschk Berl', { subject: 'Kaschk' }, true]); // jitter issue
-  // cases.push(['Kaschk Berli', { subject: 'Kaschk' }, true]); // jitter issue
+  cases.push(['Kaschk B', { subject: 'Kaschk B' }, true]);
+  cases.push(['Kaschk Be', { subject: 'Kaschk Be' }, true]);
+  cases.push(['Kaschk Ber', { subject: 'Kaschk Ber' }, true]);
+  cases.push(['Kaschk Berl', { subject: 'Kaschk Berl' }, true]);
+  cases.push(['Kaschk Berli', { subject: 'Kaschk Berli' }, true]);
```

Comment:
I'd like to test that Pelias till finds the venue with partially complete city name as before:

<img width="595" alt="Screenshot 2020-09-01 at 11 11 34" src="https://user-images.githubusercontent.com/738069/91831079-e6f4fa80-ec43-11ea-8c0a-84bf52f5d6b8.png">